src/site/content/en/blog/model-viewer/index.md

```diff
@@ -0,0 +1,160 @@
+use any web component, much less `<model-viewer>`.
+
+In this article, I will show you things that are particular to `<model-viewer>`.
+If you're interested in learning more about web components
+[webcomponents.org](https://www.webcomponents.org/) is a good place to start.
+
+## What can &lt;model-viewer> do?
+
+I'll show you a few current capabilities of `<model-viewer>`. You'll get a great
+experience today, and `<model-viewer>` will get better over time as Google adds
+new features and improves rendering quality. The examples I'm provided are just
+to give you a sense of what it does. If you want to try them there are
+installation and usage instructions in [its GitHub
+repo](https://modelviewer.dev/).
+
+### Basic 3D models
+
+Embedding a 3D model is as simple as the markup below. By
+using glb files, we've ensured that this component will work on any major
+browser.
+
+```html
+<!-- Import the component -->
+<script type="module" src="https://unpkg.com/@google/model-viewer/dist/model-viewer.min.js"></script>
+<script nomodule src="https://unpkg.com/@google/model-viewer/dist/model-viewer-legacy.js"></script>
+<!-- Use it like any other HTML element -->
+<model-viewer id="mv-demo" shadow-intensity="1" src="./spacesuit.glb"
+alt="A 3D model of an astronaut" auto-rotate camera-controls
+poster="./spacesuit.jpg"></model-viewer>
+```
+
+That code renders like this:
+
+<div class="glitch-embed-wrap" style="height: 500px; width: 100%;">
+  <iframe
+    src="https://model-viewer-shark.glitch.me/"
+    title="<model-viewer> on Glitch"
+    allow="geolocation; microphone; camera; midi; vr; encrypted-media;
+clipboard; clipboard-read; clipboard-write"
+    style="height: 100%; width: 100%; border: 0;">
+  </iframe>
+</div>
+
+### Adding motion and interactivity
+
+The `auto-rotate` and `camera-controls` attributes provide motion and user
+control. Those aren't the only possible attributes. See the documentation for [a
+complete list of](https://modelviewer.dev/).
+
+```html
+<model-viewer src="assets/Astronaut.glb" auto-rotate camera-controls>
+```
+
+### Poster image/delayed loading
+
+Some 3D models can be very large, so you might want to hold off loading them
+until the user has requested the model. For this, the component has a built-in
+means of delaying loading until the user wants it. That's what the `poster`
+attribute does.
+
+```html
+<model-viewer src="assets/Astronaut.glb" controls auto-rotate
+poster="./spacesuit.jpg">
+```
+
+To show your users that it's a 3D model, and not just an image, you can provide
+some preload animation by using script to switch between multiple posters.
+
+```html
+<model-viewer id="toggle-poster" src="assets/Astronaut.glb" controls
+auto-rotate poster="assets/poster2.png"></model-viewer>
+<script>
+    const posters = ['poster.png', 'poster2.png'];
+    let i = 0;
+    setInterval(() =>
+        $('#toggle-poster').setAttribute('poster', `assets/${posters[i++ %
+2]}`), 2000);
```

Comment:
```suggestion
        document.querySelector('#toggle-poster').setAttribute('poster', 
            `assets/${posters[i++ % 2]}`), 2000);
```
Replaces jQuery with vanilla JS and makes the indentation aligned with Google's JS style guide